src/keys_handler.rs

```diff
@@ -2,26 +2,207 @@
 // Copyright 2021 Keylime Authors
 
 use actix_web::{web, HttpResponse, Responder};
+use log::*;
 use serde::Deserialize;
 
+use json::parse;
+use openssl::{
+    aes::AesKey,
+    encrypt::{Decrypter, Encrypter},
+    pkey::{PKey, Private, Public},
+    rsa::{Padding, Rsa},
+    symm::{decrypt_aead, Cipher},
+};
+
+use crate::{error::Error, QuoteData};
+
+// This value comes from and must match the Python codebase
```

Comment:
I think a better reference for this constant could be [FIPS 197](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.197.pdf), Introduction: `This standard specifies the Rijndael algorithm ([3] and [4]), a symmetric block cipher that can process data blocks of 128 bits, ...`. (128 bits / 8 bits-per-byte gives you 16)

Reply:
Actually, a better approach: get rid of this constant. Use [Cipher.block_size](https://docs.rs/openssl/0.10.34/openssl/symm/struct.Cipher.html#method.block_size).